From 0e10ffc838980e0a61dec17ad5102cd5fcdba99e Mon Sep 17 00:00:00 2001
From: Iain Sandoe <iain@sandoe.co.uk>
Date: Tue, 8 Sep 2020 19:52:10 +0100
Subject: [PATCH] testsuite, Darwin, Arm64 : Adjust for revised section
 spellings.

We fixed some section names to change __common -> __bss which
affects this testcase.

(cherry picked from commit bb68974eb67d09ee1c46ea4518dc29785fe7ff50)
Signed-off-by: Kirill A. Korinsky <kirill@korins.ky>
---
 gcc/testsuite/g++.dg/abi/aarch64_guard1.C | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git gcc/testsuite/g++.dg/abi/aarch64_guard1.C gcc/testsuite/g++.dg/abi/aarch64_guard1.C
index 715750ae019..52be32decc6 100644
--- gcc/testsuite/g++.dg/abi/aarch64_guard1.C
+++ gcc/testsuite/g++.dg/abi/aarch64_guard1.C
@@ -13,5 +13,5 @@ int *foo ()
 }
 
 // { dg-final { scan-assembler _ZGVZ3foovE1x,8,8 { target { ! *-*-darwin* } } } }
-// { dg-final { scan-assembler __DATA,__common,__ZGVZ3foovE1x,8,3 { target *-*-darwin* } } }
+// { dg-final { scan-assembler __DATA,__bss,__ZGVZ3foovE1x,8,3 { target *-*-darwin* } } }
 // { dg-final { scan-tree-dump "& 1" "original" } }
-- 
2.42.1

